package/MDAnalysis/topology/PDBParser.py

```diff
@@ -87,6 +88,75 @@ def float_or_default(val, default):
     except ValueError:
         return default
 
+digits_upper = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+digits_lower = digits_upper.lower()
+digits_upper_values = dict([pair for pair in zip(digits_upper, range(36))])
+digits_lower_values = dict([pair for pair in zip(digits_lower, range(36))])
```

Comment:
These are constants in the global name space, they should be `CAPITALIZED_WITH_UNDERSCORES`.

Reply:
Resolved in https://github.com/MDAnalysis/mdanalysis/pull/1978/commits/02bf546b3d16f3503732e495b06c8915fda42b6f